src/main.py

```diff
@@ -2716,6 +2717,15 @@ def post_all(self, o):
         email = post["email"]
         signature = post["signature"]
         quicklinks = post["quicklinksid"]
+        rawfavouritereports = post["favouritereportsid"]
+        favouritereports = ""
+        for reportid in rawfavouritereports.split(","):
+            for report in asm3.reports.get_reports(o.dbo):
+                if report.ID == int(reportid):
+                    reporttitle = report.TITLE
+            if favouritereports != "":
+                favouritereports += ","
+            favouritereports += reportid + "=" + reporttitle
```

Comment:
concatenation! do something like quickreportscfg.append( f"{reportid}={reporttitle}" ) instead

Reply:
I did see this one coming :)